src/data-workspace/title-bar/title-bar.js

```diff
@@ -0,0 +1,78 @@
+import { useConfig } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import { IconInfo16, IconDimensionDataSet16, Tooltip } from '@dhis2/ui'
+import { formatDistance } from 'date-fns'
```

Comment:
Ah. Okay. Yeah. I was trying to not use moment because the maintainers themselves more or less say that you shouldn't (https://momentjs.com/docs/#/-project-status/), but I guess if it's already bundled in the app shell, I guess it doesn't make sense to import a different library in addition

Reply:
Implemented